src/CMakeLists.txt

```diff
@@ -12,8 +12,12 @@ add_subdirectory(storage)
 add_subdirectory(transaction)
 
 add_library(kuzu STATIC ${ALL_OBJECT_FILES})
-target_link_libraries(kuzu PUBLIC antlr4_cypher antlr4_runtime utf8proc re2
-        parquet_lib arrow_lib arrow_deps Threads::Threads)
-target_include_directories(
-        kuzu PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+target_link_libraries(kuzu
+        PUBLIC antlr4_cypher antlr4_runtime utf8proc re2 parquet_lib arrow_lib arrow_deps Threads::Threads)
+target_include_directories(kuzu
+        PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include> $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+add_library(kuzu_shared SHARED ${ALL_OBJECT_FILES})
+target_link_libraries(kuzu_shared
```

Comment:
We should add `set_target_properties(kuzu_shared PROPERTIES OUTPUT_NAME kuzu)` so that the shared lib is also called `libkuzu` instead of `libkuzu_shared`

Reply:
Done.